content/en/docs/install-operate/deployment.md

```diff
@@ -1,13 +1,27 @@
 ---
 title: Deployment
-description: Installing Falco on a Cluster
+description: Installing Falco on a Kubernetes Cluster
 aliases: [/docs/getting-started/deployment]
-weight: 40
+weight: 45
 ---
 
+The guides - [Download](/docs/install-operate/download/), [Install](/docs/install-operate/installation/), [Running](/docs/install-operate/running/) - focused on the systemd service deployment method, detailing how to primarily install and run Falco directly on the host. Additionally, these guides explored the alternative options of installing and running Falco within a container, with a driver installed on the underlying host.
+
+The container deployment option is particularly enticing for Kubernetes settings, which will be discussed in more detail now. 
+
+{{% pageinfo color="info" %}}
+
+Falco is designed to run once per instance of the Linux kernel, regardless of whether the deployment is on a bare-metal host or a virtual machine (VM). In third-party managed Kubernetes platforms (EKS, GKS, AKS, etc.), the latter option (VM) is prevalent.
```

Comment:
@nate-double-u incorporated your suggestions and added a clarifying sentence on top plus formatting enhancements. Mind checking once more if ok? Thanks!

Reply:
I think that looks good to me 🙂